source/js/weekly_navigation.js

```diff
@@ -43,12 +43,129 @@ const sideBar = document.querySelector('#mySideBar ul');
 // the function content is at the bottom of the file
 let sideBarClick = function sideBarClickedTemp() {};
 
+
+ // Test Events
+ let test_events = {
```

Comment:
Codacy has a fix for the issue: ['test_events' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
    const test_events = {
```